src/OrchardCore/OrchardCore.ContentManagement/Script/HtmlScriptSanitizer.cs

```diff
@@ -0,0 +1,14 @@
+using Ganss.XSS;
+
+namespace OrchardCore.ContentManagement.Script
+{
+    public class HtmlScriptSanitizer : IHtmlScriptSanitizer
+    {
+        public string Sanitize(string html)
+        {
+            var sanitizer = new HtmlSanitizer();
+
```

Comment:
I believe this is threadsafe and reusable, so can be created once as a singleton.

(we'll probably need some options for it, but later...)

Reply:
Sorry bad advice. as a static instance inside this class not a singleton